subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/FileTimeStampInspector.java

```diff
@@ -92,4 +92,12 @@ public boolean timestampCanBeUsedToDetectFileChange(String file, long timestamp)
         // Do not use a timestamp that is the same as the end of the last build or the start of this build
         return timestamp != lastBuildTimestamp;
     }
+
+    private long readBuildTimestampFromMarkerFile() {
+        return readTimestampFrom(markerFile);
+    }
+
+    private long readTimestampFrom(File file) {
```

Comment:
Nit: initially when I read the method name I thought "the file contains a timestamp". Maybe `timestampOf` would be clearer.

Reply:
Renamed to `timestampOf`.